### DISCUSSION TITLE
LineSeries points Drag Drop Functionality

### DISCUSSION THREAD

**Discussion by rajeshytech** [Live-Charts/LiveCharts2#1225] [Q&A]
Hi ,

I am using LivechartCore in wpf application. I have LineSeries and i want to Drag & Drop from the points and accordingly the the point value should update , Could you please help me whether Drag Drop functionality is there or not for LineSeries points values. If yes please help me with some samples. Please find the attached image below .

Thanks in advance.

![MicrosoftTeams-image](https://github.com/beto-rodriguez/LiveCharts2/assets/19607597/8505e72c-a268-4eae-a49b-960a7004e3d9)

**Comment by Rkjena10** [comment]
Kindly help me on  above requirement. I have also same kind of requirements. Thanks in advance

**Comment by Rkjena10** [comment]
Could you please help me on drag drop functionality on specific point in multiline series. I couldn't progress since this is client requirement. Please help me on this.
